SergioMendozaParra/August 26/Hackerrank/Java_Inheritance_I/Main.java

```diff
@@ -0,0 +1,10 @@
+package javaInheritance1;
```

Comment:
NO uppercase in package name please

Reply:
Sorry, I forgot not to put them in lowercases.